app/controllers/shop_product_sink/webhooks_controller.rb

```diff
@@ -41,6 +41,8 @@ def resource_class
     end
 
     def shopify_resource
+    	# set site to string or ActiveResource raises on call to .prefix on nil
+    	ShopifyAPI::Base.site = '' if ShopifyAPI::Base.site.nil?
```

Comment:
Is there perhaps a better place we could do this? Could you also add tests to prove that this was broken and is now fixed?